### PR TITLE
kinder: add leftover test workflows for 1.21

### PR DESCRIPTION
**Pull request by RA489**
xref https://github.com/kubernetes/kubeadm/pull/2412#pullrequestreview-626238534
test infra PR: https://github.com/kubernetes/test-infra/pull/21643

### CODE REVIEW

**Comment by RA489** [comment]
https://github.com/kubernetes/kubeadm/issues/2362

**Review by neolit123 — CHANGES_REQUESTED** [review]
skew-kubelet-1.20-on-latest.yaml is missing.

File: kinder/ci/workflows/skew-kubelet-1.20-on-1.21.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kubelet-1.20-on-latest
+  This workflow tests the proper functioning of kubeadm and control plane v1.21 against 1.20 kubelet
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kubelet-1-20-on-1-21
   config    > https://git.k8s.io/test-infra/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml
   config    > https://git.k8s.io/test-infra/testgrid/config.yaml
 vars:
-  kubeadmVersion: "{{ resolve `ci/latest` }}"
+  kubeadmVersion: "{{ resolve `ci/latest-1.21` }}"
   kubeletVersion: "{{ resolve `ci/latest-1.20` }}"
   kubernetesVersion: "{{ resolve `ci/latest` }}"
```
Comment (neolit123) [inline kinder/ci/workflows/skew-kubelet-1.20-on-1.21.yaml:10]:
```suggestion
  kubernetesVersion: "{{ resolve `ci/latest-1.21` }}"
```

File: kinder/ci/workflows/skew-kubelet-1.19-on-1.21.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
-  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kubelet-1.19-on-latest
+  This workflow tests the proper functioning of kubeadm and control plane v1.21 against 1.19 kubelet
+  test grid > https://testgrid.k8s.io/sig-cluster-lifecycle-kubeadm#kubeadm-kinder-kubelet-1-19-on-1-21
   config    > https://git.k8s.io/test-infra/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-kubelet-x-on-y.yaml
   config    > https://git.k8s.io/test-infra/testgrid/config.yaml
 vars:
-  kubeadmVersion: "{{ resolve `ci/latest` }}"
+  kubeadmVersion: "{{ resolve `ci/latest-1.21` }}"
   kubeletVersion: "{{ resolve `ci/latest-1.19` }}"
   kubernetesVersion: "{{ resolve `ci/latest` }}"
```
Comment (neolit123) [inline kinder/ci/workflows/skew-kubelet-1.19-on-1.21.yaml:10]:
```suggestion
  kubernetesVersion: "{{ resolve `ci/latest-1.21` }}"
```

**Comment by RA489** [comment]
updated the PR in https://github.com/kubernetes/kubeadm/pull/2412

**Pull request closed without merging**